docs/pages/get_started/developers/release_codemods.js

```diff
@@ -0,0 +1,310 @@
+// @flow strict
+import { type Node } from 'react';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import Page from '../../../docs-components/Page.js';
+
+export default function EslintPluginPage(): Node {
+  return (
+    <Page title="Release codemods (web)">
+      <PageHeader
+        name="Release codemods (web)"
+        description={`Run codemods to automate required code changesrelated to major releases (breaking changes) in Gestalt.
+
+Every major release in the component library comes with a codemod to facilitate the upgrade of the Gestalt dependency. Some codemods are custom built for each upgrade while others take advantage of generic codemods that only require running a codemod command with arguments.
+
+When releases include breaking changes, the specific codemod command is included in the pull request.`}
+        type="guidelines"
+      />
+      <MainSection
+        name="Generic codemods"
+        description={`Generic codemods for common tasks like renaming components or props can be found in this [directory](https://github.com/pinterest/gestalt/tree/master/packages/gestalt-codemods/generic-codemods).
+
+        throwErrorIfSpreadProps({
+
+Codemods modifying prop-values also throw errors when they encouter spread props used in Gestalt components. Spread props are opaque to codemods and require manual action.
+
+Error messages include the location (file path) and the code line of the node that caused the error to facilitate addressing the corresponding issue.`}
+      >
+        <MainSection.Subsection
+          title="renameComponent"
+          description={`
+Codemod to **RENAME** Gestalt **components**
+
+Example: transforms first code (top) into second code (bottom)
+~~~bash
```

Comment:
this should be `js`, no? (here and throughout the examples)

Reply:
YEp, didnt know that